### ISSUE TITLE
Android Studio can't resolve 'import butterknife.Views;'

### ISSUE THREAD

**Issue by IanButterworth** [inmite/android-grid-wichterle#4]
The 'Views' symbol from butterknife seems to be missing or moved to an unclear location.

Issue arises in SettingsActivity.java.

....
import butterknife.InjectView;
import butterknife.Views;
import com.squareup.otto.Subscribe;
....

**Comment by IanButterworth** [comment]
Issue can be resolved by replacing: import butterknife.Views;
with: import butterknife.ButterKnife;

and one mention of: Views.inject(this);
with: ButterKnife.inject(this);

**Comment by davidvavra** [comment]
Yes the app uses old version of ButterKnife with different API. Would you like to submit a pull request with updated ButterKnife?
